lib/data_update_scripts/20220111192031_insert_apple_connect_broadcast_message.rb

```diff
@@ -0,0 +1,18 @@
+module DataUpdateScripts
+  class InsertAppleConnectBroadcastMessage
+    def run
+      title = "Welcome Notification: apple_connect"
+      return if Broadcast.find_by title: title
+
+      message = "You're on a roll! 🎉  Do you have an Apple account? " \
+                "Consider <a href='/settings'>connecting it</a>."
+
+      Broadcast.create!(
+        title: title,
+        processed_html: message,
+        type_of: "Welcome",
+        active: true,
+      )
```

Comment:
If someone runs `rails db:seed data_updates:run`, will this fail or cause any other problems?

Reply:
the early return (find by title and return silently if present) avoids that conflict.

Reply:
Ballin. 💯 Totally missed that.

![Mic drop](https://upload.wikimedia.org/wikipedia/commons/1/15/Barack_Obama_drops_the_mic.gif)

Reply:
```
 bundle exec rails db:drop db:create db:schema:load db:seed data_updates:run
```

This succeeded without error - and insert apple broadcast message was marked as finished in the logs :relieved: